.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
why is this file added

Reply:
I have not added this file but it's already removed.